src/main/java/Deadlines.java

```diff
@@ -0,0 +1,22 @@
+
+/**
+ * Tasks that need to be done before a specific time/date.
+ */
+public class Deadlines extends Task {
+
+    private String by;
```

Comment:
Perhaps this variable name for the date/time for the deadline too short? Perhaps make it a bit longer and more explicit like dateOfDeadline?